pkg/controllers/disruption/consolidation_test.go

```diff
@@ -3677,42 +3834,34 @@ var _ = Describe("Consolidation", func() {
 					},
 				},
 				Status: v1beta1.NodeClaimStatus{
-					ProviderID:  test.RandomProviderID(),
 					Allocatable: map[v1.ResourceName]resource.Quantity{v1.ResourceCPU: resource.MustParse("1")},
 				},
 			})
-			zone2NodeClaim, zone2Node = test.NodeClaimAndNode(v1beta1.NodeClaim{
-				ObjectMeta: metav1.ObjectMeta{
-					Labels: map[string]string{
-						v1beta1.NodePoolLabelKey:     nodePool.Name,
-						v1.LabelTopologyZone:         "test-zone-2",
-						v1.LabelInstanceTypeStable:   testZone2Instance.Name,
-						v1beta1.CapacityTypeLabelKey: testZone2Instance.Offerings[0].CapacityType,
-					},
-				},
-				Status: v1beta1.NodeClaimStatus{
-					ProviderID:  test.RandomProviderID(),
-					Allocatable: map[v1.ResourceName]resource.Quantity{v1.ResourceCPU: resource.MustParse("1")},
-				},
+			nodeClaims[1].Labels = lo.Assign(nodeClaims[0].Labels, map[string]string{
```

Comment:
i think these should be the same index?
```suggestion
			nodeClaims[1].Labels = lo.Assign(nodeClaims[1].Labels, map[string]string{
```

Reply:
Done.